src/components/gameBoard.scss

```diff
@@ -1,27 +1,27 @@
 .GameBoard {
-  .GameBoard__gameInfoControl {
+  &__gameInfo {
     h3 {
       margin: 15px 0;
     }
 
     h4 {
       margin: 15px 0;
     }
+  }
 
-    .GameBoard__restartGameButton {
-      cursor: pointer;
-    }
+  &__restartGameButton {
+    cursor: pointer;
   }
-}
 
-.GameBoard__board {
-  margin-top: 20px;
-  display: flex;
-  width: 460px;
-  justify-content: center;
-  flex-wrap: wrap;
+  &__board {
+    margin-top: 20px;
+    display: flex;
+    width: 460px;
+    justify-content: center;
+    flex-wrap: wrap;
 
-  &.disabled {
-    pointer-events: none;
+    &.disabled {
```

Comment:
This is acceptable because `disabled` is more like a global usage. As you can see there's really no absolute rule of how things should be done.